style.css

```diff
@@ -0,0 +1,78 @@
+/* nejsme vcerejsi, pouzivame moderni box model */
+
+html {
+    box-sizing: border-box;
+}
+
+*,
+ :after,
+ :before {
+    box-sizing: inherit;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 14px;
+}
+
+
+/* ---------------------------------------------------- */
+
+.zeny {
+    display: flex;
+    background-color: rgb(211, 211, 211);
+    border-radius: 30px;
+    max-width: 1300px;
+    margin: 2rem;
+}
+
+.hanka {
+    display: block;

```

Comment:
Tady jde hezky vidět, že by ti stačila 1 třída, máš ve všech úplně to samé 🙂

Reply:
Jinak `display: block; `vůbec nemusíš uvádět - je to defaultní (výchozí) hodnota, takže pokud ji nenapíšeš, i tak se aplikuje. Tady bych se ani nebála nastavit jednotivým položkám (holkám) display flex, aby se vůči svému obsahu (fotka, popis...) chovaly jako flexbox. To nám pak trochu usnandní práci s dalším pozicováním, např. kdybychom chtěly, aby ikonky sociálních sítí byly vždy úplně dole 🙂